collect_app/src/main/java/org/odk/collect/android/tasks/InstanceGoogleSheetsUploader.java

```diff
@@ -391,10 +428,47 @@ private void createSheetsIfNeeded(TreeElement element) throws UploadException {
         return answers;
     }
 
-    private List<Object> getColumnTitles(TreeElement element) {
+    /**
+     * Strip the Altitude and Accuracy from a location String and add them as separate column if
+     * the column titles exist.
+     *
+     * @param answers - HashMap of current Answers
+     * @param elementTitle - Current header title
+     * @param answer - String that holds the Lat/Long/Altitude/Accuracy
+     */
+    private void parseAltitudeAndAccuracy(@NonNull Map<String, String> answers, @NonNull List<Object> columnTitles, @NonNull String elementTitle, @NonNull String answer) {
```

Comment:
I think a better name would be `parseGeopoint` since this is also separating out the lat/long. How about not passing in `answers` and returning a new map from column title to value that gets added to `answers` by the caller (`getAnswers`)? I know several other methods in this class mutate their parameters but it makes the code harder to read and in this case I think it's easy to avoid.

Reply:
Agreed - parseGeopoint is a better name - Renamed.

Passing back a Map is cleaner, i'll do a refactor - I should be able to use something like answers.putAll(parseGeopoint(...));